docfiles/script.html

```diff
@@ -27,6 +27,7 @@ <h2 class="ui item header no-select">@title@</h2>
                         <a id="editCodeButton" href="/@versionsuff@#pub:@id@" class="ui mini button landscape only" role="menuitem">Edit Code</a>
                     </div>
                     <div class="page-header-item no-grow">
+                        <button id="multiplayer-share-button" class="ui mini default button" role="menuitem" style="display: none">Play online with friends</button>
```

Comment:
"Host Game" is the terminology used in the multiplayer

![image](https://user-images.githubusercontent.com/6107272/199290856-676d0517-88b2-405a-bc19-9f5f6fe2c1f8.png)